### PR TITLE
Release notes for MATE 1.24

### PR DESCRIPTION
**Pull request by raveit65**
This is a template for 1.24 release copied from 1.22 release notes commit.
Feel free to add more real content.

### CODE REVIEW

**Comment by lukefromdc** [comment]
These are coming along nicely

**Comment by sc0w** [comment]
is not better to do first the 1.24 commits to all repositories?

**Comment by raveit65** [comment]
Yes, this PR should only merged when all release commits are done.
But we can prepare the release notes simultaneously, the details can simply taken from 1.23 release notes.
As you can see I filtered the commits a bit for atril, caja, etc.
I don`t think that the complete changelog belongs into the release notes like we did for previous releases. Nobody will read the whole changelog ;)
After all someone from our native English speakers will hopefully write the main text for the release notes.
I will start with commiting the release commits these week, but without release tag, because the release tag will trigger travis to build the tarballs.
**So, please do not push any commits to repos when you see an existing 1.24 commit.**

**Comment by raveit65** [comment]
> I propose the changes below for the atril section:

Done.

**Comment by raveit65** [comment]
@rbuj 
Did i override changes from you?

**Comment by rbuj** [comment]
@raveit65 
Don't worry, next I wrote a comment proposing the changes. I think also that `Require WebKitGTK-4.0 2.6.0 or newer` is better than `Require WebKitGTK-4.0 >= 2.6.0`, `Require GLib 2.54.0 or newer` and so on.

**Comment by raveit65** [comment]
Feel free to correct my first raw content.
At the moment i want only to filter out relevant commit from changelog for all repos, without thinking to much about it.
It's also possible that i missed an important change.

**Comment by rbuj** [comment]
```diff
-![about](/assets/img/blog/about-mate-1.24.png)
+![MATE About dialog](/assets/img/blog/about-mate-1.24.png){ width=50% }
```

**Comment by sc0w** [comment]
I think the "about dialog" is too big, in 1.22 we see the dialog smaller

see the differences:

https://preview.mate-desktop.dev/blog/2020-02-04-mate-1-24-released/

https://preview.mate-desktop.dev/blog/2019-03-18-mate-1-22-released/

**Comment by raveit65** [comment]
@rbuj @sc0w 
mate-about-dialog image size is updated, OK?

**Comment by rbuj** [comment]
It looks great in the preview!

**Comment by raveit65** [comment]
@mate-desktop/core-team 
I finished today with adding a raw content for release notes.
Feel free to correct mistakes by me, or add features which i have forgotten to add.
At least someone with with better English like me should write the final release text based on the content of the package details, please.......
This is missing!!!

**Comment by rbuj** [comment]
It's a minor edit, the contents have been selected very well, nice job!

**Comment by vkareh** [comment]
Thanks for starting this @raveit65 - I've added release highlights at the top. Let me know if it all looks fine. I can change/remove/add anything else needed too.

File: blog/20200210-mate-1-24-released.md [hunk 1]
```diff
@@ -380,190 +378,189 @@ Complete changelog at <https://github.com/mate-desktop/mate-notification-daemon/
...
+* xrandr: Add bigger icon sizes for tray-applet
 * Hide MATE Settings Daemon from Startup Applications capplet
-* migrate from intltool to gettext
 * Add remaining peripheral schemas from mate-desktop
 * mouse: Add acceleration profiles
 * Change media-keys volume-step default from 6 to 5 in the gschema
-* plugins: drop libXxf86 dependency
-* plugins housekeeping: optimization code
+* plugins: Drop libifenici dependency
+* plugins housekeeping: Code optimization
```
Comment (raveit65) [inline blog/20200210-mate-1-24-released.md]: Is this a typo? libXxf86 --> libifenici

**Comment by raveit65** [comment]
Main Text and changes for details are looking fine, except the little typo with m-s-d, or wasn't it a typo?

```
The Time And Date Manager app has been rewritten with a much nicer interface.
```
Time-Admin application was added in June last year. See https://github.com/mate-desktop/mate-control-center/pull/456
I think we should announce it as new application and not as a rewrite.
We never had a old time management application.

Any way, the release notes includes a lot of news and they are not overloaded and easy to read.
Thanks for the update.

File: blog/20200210-mate-1-24-released.md (hunk 1 above)
Comment (rbuj) [reply blog/20200210-mate-1-24-released.md]: I apologize for the inconvenience caused by the Catalan autocorrect.

Comment (raveit65) [reply blog/20200210-mate-1-24-released.md]: yeah, autocorrect is evil :)

**Comment by raveit65** [comment]
I will start today with preparing tarballs. When this is done i will update release date in a last step before merging.

**Comment by raveit65** [comment]
@vkareh @rbuj 
I think we can add 2 more features to highlights.
- pluma , support for drawing spaces/tabs/newlines in docs with an option to enable/disable it.
- session-manager, we do not show hidden startup applications in session-properties as default , or not? Also with an option to show this entries.

Thoughts?

**Comment by rbuj** [comment]
Pluma no longer has to envy anything from other complex editors since it can now show the formatting marks.

**Comment by rbuj** [comment]
We did it again, the MATE desktop environment is easier to use than before after the user starts the session. Do you want to hide applications startup? Now you can set which applications to show on startup.

**Comment by raveit65** [comment]
@rbuj 
offtopic
do you like to join our video meeting about wayland at  10am EST ?
https://github.com/orgs/mate-desktop/teams/core-team/discussions/19
Link is https://meet.jit.si/mate-on-wayland

**Comment by raveit65** [comment]
@rbuj 
Nice text, i like it.

**Comment by raveit65** [comment]
+1 for using BlueSubmarine theme for Catalan translated page :)

**Comment by sc0w** [comment]
@vkareh 

> Además, pluma ya no tiene nada que enviarle a otros editores de texto más complejos, ya que ahora puede mostrar marcas de formateo.

Do you mean "envidiarle" instead "enviarle" ?

**Comment by sc0w** [comment]
when the branch 1.24 is done we can use `https://github.com/mate-desktop/.../commits/1.24/` instead `https://github.com/mate-desktop/.../commits/master/` in the list of changes

**Comment by vkareh** [comment]
Envidiarle! Thanks for catching the typo :) I'll fix it tonight

**Comment by lukefromdc** [comment]
While we are discussing the 1.24 release, note that we have an error in mate-desktop 1.24.0:
https://github.com/mate-desktop/mate-desktop/issues/429
meson builds show wrong version in mate-about, still showing 1.23.3 instead of 1.24.0

**Comment by raveit65** [comment]
OMG, https://github.com/mate-desktop/mate-desktop/commit/10c192c904f9889153e2788bef9b2520b3068864

**Comment by raveit65** [comment]
I squashed some commits where it was possible.

**Comment by raveit65** [comment]
@rbuj @vkareh 
I added
```
* i18n: All applications are migrated from intltools to gettext now
```
to highlights, to honour the great and huge work by @yetist .
Please update your language pages.

**Pull request merged**